src/zajecia9/zadanie2/Rent/RentableCar.java

```diff
@@ -0,0 +1,55 @@
+package zajecia9.zadanie2.Rent;
+
+import zajecia9.zadanie2.Vehicles.Car;
+
+public class RentableCar extends Car implements Rentable {
+    public static Person NONE = null;
+    private Person person;
+    private boolean isRented;
+    private String id;
```

Comment:
to pole nalezy do klasy `Person`

Reply:
wlasnie tu mialam dylemat, nie bylo to dla mnie jasne z tresci zadania